wicket-examples/src/main/java/org/apache/wicket/examples/ajax/builtin/AjaxApplication.java

```diff
@@ -44,6 +46,11 @@ protected void init()
 
 		getDebugSettings().setAjaxDebugModeEnabled(true);
 		
+		setHeaderResponseDecorator(response -> {
```

Comment:
this can be simplified to be
```
tester.getApplication().setHeaderResponseDecorator(response -> new ResourceAggregator(new JavaScriptDeferHeaderResponse(response)));
```